tools/Test/SmokeTest/RmCoreSmokeTests.ps1

```diff
@@ -37,83 +37,101 @@ $resourceGroupName = "azpssmokerg$randomValue"
 $storageAccountName = "azpssmokesa$randomValue"
 
 $resourceSetUpCommands=@(
-    @{Name = "Az.Resources";                  Command = {New-AzureRmResourceGroup -Name $resourceGroupName -Location westus -ErrorAction Stop}}
+    @{Name = "Az.Resources";                  Command = {New-AzResourceGroup -Name $resourceGroupName -Location westus -ErrorAction Stop}}
 )
 $resourceCleanUpCommands = @(
-    @{Name = "Az.Storage [Cleanup]";          Command = {Remove-AzureRmStorageAccount -Name $storageAccountName -ResourceGroupName $resourceGroupName -Force -ErrorAction Stop}},
-    @{Name = "Az.Resources [Cleanup]";        Command = {Remove-AzureRmResourceGroup -Name $resourceGroupName -Force -ErrorAction Stop}}
+    @{Name = "Az.Storage [Cleanup]";          Command = {
+                                                          $loopLimit = 0
+                                                          $flag = $false
+                                                          do {
+                                                            try {
+                                                              $loopLimit ++
+                                                              Remove-AzStorageAccount -Name $storageAccountName -ResourceGroupName $resourceGroupName -Force -ErrorAction Stop
+                                                              $flag = $false
+                                                            }
+                                                            catch {
+                                                              $flag = $true
+                                                              Start-Sleep -Seconds 30
+                                                              if ($loopLimit -gt 30)
+                                                              {
+                                                                throw $_.Exception
+                                                              }
+                                                            }
+                                                          } while ($flag -and ($loopLimit -le 30))
+                                                        }},
+    @{Name = "Az.Resources [Cleanup]";        Command = {Remove-AzResourceGroup -Name $resourceGroupName -Force -ErrorAction Stop}}
```

Comment:
Please add similar retry-logic for Az.Resources [Cleanup], thanks

Reply:
Done